prevent-broker-scale-down-if-contains-partition .md

```diff
@@ -0,0 +1,45 @@
+
+# Prevent scale down of the brokers if they contain replicas
```

Comment:
as already done for the auto-rebalancing proposal, let's try to use `partition replicas` instead of just `replicas` everywhere.

Reply:
Sure on it